src/bp/ui-shared/src/ConfirmDialog/index.tsx

```diff
@@ -23,7 +23,7 @@ const ConfirmDialogComponent: FC<ConfirmDialogProps> = props => {
   }
 
   return (
-    <BaseDialog icon="warning-sign" usePortal={false} isOpen onClose={onDecline}>
+    <BaseDialog icon="warning-sign" usePortal={false} isOpen onClose={onDecline} size={props.size}>
```

Comment:
I would just hardcode the size to "sm" since the confirmDialogs are meant to be as lean and fewer options as possible when adding them somewhere (basically the equivalent of window.confirm). Don't forget to remove the size option you added in the dialogs 😬 

```suggestion
    <BaseDialog icon="warning-sign" usePortal={false} isOpen onClose={onDecline} size="sm">    
```